classes/cargo.bbclass

```diff
@@ -59,14 +75,13 @@ export RUST_CFLAGS = "${HOST_CC_ARCH}${TOOLCHAIN_OPTIONS} ${CFLAGS}"
 export RUST_BUILD_CC = "${CCACHE}${BUILD_PREFIX}gcc"
 export RUST_BUILD_CFLAGS = "${BUILD_CC_ARCH} ${BUILD_CFLAGS}"
 
-RUSTFLAGS ??= ""
-export CARGO_BUILD_FLAGS = "-v --target ${HOST_SYS} --release"
+CARGO_BUILD_FLAGS = "-v --target ${HOST_SYS} --release"
```

Comment:
This could be added in a separate set of changes, but I have needed to modify the cargo build flags for some projects in the past and having something like `EXTRA_CARGO_FLAGS` is useful.  Some of the things we do here would seem useful to mimic (meta-rust-bin also deals with debug/release builds): https://github.com/rust-embedded/meta-rust-bin/blob/master/classes/cargo.bbclass